### DISCUSSION TITLE
handle basic auth with http git bare remote repo

### DISCUSSION THREAD

**Discussion by noor-tg** [Vinzent03/obsidian-git#796] [Q&A]
I use local http git bare repo like `http://192.168.1.10:8080/brain.git` . and the git repo is protected by basic auth password.
I want to git & push to git repo from this plugin commands. but I have a problem with making it work with http basic authentication method.

so the question is like in the title . how to do it ?

**Comment by Vinzent03** [comment]
Please try again with the new version v.2.30.0 everything should work like with every other git remote provider.

**Reply by noor-tg** [reply]
I started to use git-lfs . And I know isomorphic-git does not support it. So I use git from termux for now.

But thanks for your great effort with this plugin.
